stats/stats_test.go

```diff
@@ -786,8 +884,14 @@ func checkConnEnd(t *testing.T, d *gotData) {
 	st.IsClient() // TODO remove this.
 }
 
+type event struct {
+	eventType string
+	timestamp time.Time
```

Comment:
The timestamp was included for debugging purposes in case of test failures, since it could help identify delays or other issues. If you feel this isn’t valuable, I can to remove it for simplicity.